wiki/Contests/VMC/1/en.md

```diff
@@ -0,0 +1,84 @@
+---
+tags:
+  - VMC1
+  - VMC
+---
+
+# Vocaloid Mapping Contest
+
+![](/wiki/Contests/VMC/1/img/banner.png)
+
+The [**Vocaloid Mapping Contest (VMC)**](/wiki/Contests/VMC) is all about finding and promoting VOCALOID, UTAU, and other vocal synth songs and artists that have fallen through the cracks. Mappers were tasked with creating the best beatmap possible, and then having them judged by some of the most prominent VOCALOID mappers in the community. 
+
+In all subsequent VMC's there will be an associated theme. This could include songs based around a song theme, such as an artist, genre, voicebank, or a map theme/limitation. We will never pick a song that has been ranked in osu! within the last ~10 years, and we always try to pick songs that are particularly relevant and timeless.
+
```

Comment:
- wrong register; don't use "we"
- there's no point in including this type of announcement in this article, especially since this iteration doesn't seem to even have a theme

```suggestion
```